crates/router/src/core/user_role.rs

```diff
@@ -871,6 +871,12 @@ pub async fn list_invitations_for_user(
                 .attach_printable("Failed to compute entity id and type")?;
 
             match entity_type {
+                EntityType::Tenant => {
+                    return Err(UserErrors::InvalidRoleOperationWithMessage(
+                        "Tenant roles are not allowed for this operation".to_string(),
+                    )
+                    .into());
+                }
```

Comment:
Should we throw error here? Is dashboard handling this?

Reply:
The query above it will fetch 0 user_roles for entity type tenant where status is `invitation_sent`, so we should throw error here instead of sending empty vector, if we have such user role.

Reply:
status should be `500`